docs/cosmology/dev.rst

```diff
@@ -116,10 +116,10 @@ the definition of |FLRW|.
                 raise ValueError("baryonic density can not be larger than total matter density.")
             return value
 
-First note that all the parameters are also arguments in ``__init__``. This is
-not strictly necessary, but is good practice. If the parameter has units (and
-related equivalencies) these must be specified on the Parameter, as seen in
-:attr:`~astropy.cosmology.FLRW.H0` and :attr:`~astropy.cosmology.FLRW.m_nu`.
```

Comment:
Hmm, this seems pretty bad: you cannot actually link to the description of the parameter any more? Does that also mean `FLRW.H0?` does not work any more? That is terrible!

Reply:
This PR does not do that. That's for a future PR making `Cosmology` a dataclass.

Reply:
But this PR does shift the priority to ``.parameters``, away from the descriptor objects.

Reply:
OK, the shift itself makes sense to me. But then the docs should be good - let me take that out of to the main thread.

Reply:
👍.
Resolving this thread then.